README.md

```diff
@@ -1 +1,139 @@
-# hello-world
\ No newline at end of file
+# hello-world
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello World] <a name="about-project"></a>
+
+Hello World is a project done in the Microverse program. It shows a header with the text "Hello Microverse".
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[display h1 tag]**
+- **[to display Hello MIcrovers]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+## Prerequisites
+
+In order to run this project you need:
+
+> Computer with internet and
+> Visual Studio Code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+1. Clone this repository or download the Zip folder:
+
+**`git clone https://github.com/MohamedHNoor/Awesome-Books.git`**
+
+2. Navigate to the location of the folder in your machine:
+
+**`you@your-Pc-name:~$ cd <folder>`**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
```

Comment:
- [ ] Since you are the only contributor to this project please make sure that you change `Authors` to `Author` to prevent typo in your README file. :pray: